docs/content/PageHeader.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: PageHeader
+componentId: page_header
+status: Draft
+description: This component can be used independently or inside PageLayout.Header. It helps provide structure and responsive aspects to the Header component.
+# source: https://github.com/primer/react/tree/main/src/PageHeader
+# storybook: https://primer.style/react/storybook?path=/story/layout-pageheader--default
+---
+
+```js
+import {PageHeader} from '@primer/react'
+```
+
+### Default
+
+```jsx live
+<PageHeader>
+  {/* Visible only in narrow viewport*/}
+  <PageHeader.ContextNav>
+    {/* Always renders with a backbutton and is left aligned*/}
+    <PageHeader.ParentLink href="/code">Code</PageHeader.ParentLink>
+  </PageHeader.ContextNav>
+  {/* Visible in all viewports */}
+  <PageHeader.TitleArea>
+    <PageHeader.Title>Branches</PageHeader.Title>
+    <PageHeader.TitleActions>
+      {/* Visible only in narrow viewport*/}
+      <Stack direction={{narrow: 'inline', regular: 'none'}}>
+        <Button variant="primary">New Branch</Button>
+      </Stack>
+      {/* Visible only in regular viewport*/}
+      <Stack direction={{narrow: 'none', regular: 'inline'}}>
+        <Button variant="primary">New</Button>
+      </Stack>
+      {/* Visible in all viewports */}
+      <IconButton aria-label="More actions" icon={KebabHorizontalIcon} />
+    </PageHeader.TitleActions>
+  </PageHeader.TitleArea>
+</PageHeader>
+```
+
+### With Context Actions and Breadcrumbs
+
+<PageHeader>
```

Comment:
This example and the following examples don't have the `jsx live` codefence args. Is this intentional?

Reply:
nope! it was a mistake. Will fix this 👍